### DISCUSSION TITLE
阿里网盘下载提示This XML file does not appear to have any style information associated with it. The document tree is shown below.

### DISCUSSION THREAD

**Discussion by wanshiwui** [AlistGo/alist#236] [Q&A] [answered]
网盘类型：阿里网盘
问题描述：可以正常列出阿里文件列表，但是文件点击下载提示
![image](https://user-images.githubusercontent.com/39075368/146663896-59f7bef2-631c-429c-ba81-142ffc30d360.png)
`This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>InvalidArgument</Code>
<Message>At least one of AdditionalHeaders is not in request headers.</Message>
<RequestId>61BEB4D3937D8238345F1F8D</RequestId>
<HostId>bj29.cn-beijing.data.alicloudccp.com</HostId>
</Error>`
如果打开设置里面的web proxy代理 可以下载文件

**Comment by xhofe** [comment, marked as answer]
https://alist-doc.nn.ci/docs/driver/aliyundrive

**Reply by wanshiwui** [reply]
感谢大佬，更换为移动端后解决了问题
